Dockerfile-ray-node

```
@@ -11,7 +11,7 @@ COPY --chown=$RAY_UID:$RAY_UID ./client ./qs
 WORKDIR /qs
 # TODO: certifi version pinned to 2023.7.22. More info in #870
 # TODO: pyarrow version pinned to 14.0.1
-RUN pip install . --no-cache-dir &&\
+RUN pip install -r requirements-lock.txt --no-cache-dir &&\
     pip install --no-cache-dir pyarrow==14.0.1 &&\
```

I think you also need to install quantum-serverless from source:

```suggestion
RUN pip install -r requirements-lock.txt --no-cache-dir &&\
    pip install --no-cache . &&\
    pip install --no-cache-dir pyarrow==14.0.1 &&\
```

---

Maybe you are right, let's see!